src/test/resources/com/digitalasset/testing/cucumber.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Cucumber support in the Functional Testing library
+  Cucumber support needs to work.
+
+  Background:
+    Given Sandbox is started with DAR "src/test/resources/ping-pong.dar"
+    | Alice | Bob |
+
+  Scenario: A contract can be created and observed.
+    Given "Bob" creates contract "PingPong:Ping" using values
+    | Bob | Alice | 3 |
```

Comment:
Field names should be visible in the feature files. I suggest the following format:

```
| Field1 | Value1 |
| Field2 | Value2 |
...
```

Reply:
What about
```
| FieldName1 | FieldName2 | ...
| Value1         | Value2          | ...
```
(and accepting the 1-liner format as well)?
It is a personal preference but for me this "row layout" of values looks more readable (and similar to DAML).

Reply:
For me it's also a bit easier to read the header + body suggested by Gabor.
Either way, Gabor, could you pls extend this test with other another row to see that this is an actual table?
e.g.:
```
| sender | receiver | count |
| Bob | Alice | 3 |
| Alice | Bob | 4 |
```

Reply:
In tests I think the number of "rows" are limited, but the number of fields can be large. It'd be more legible if one don't have to scroll horizontally.

Reply:
Oh, that makes sense, ok.

Reply:
@richardkapolnai-da it is not a table, it needs to be a list of arguments (there is a check for that in the code as well)

Reply:
Could it be a table though? It'd make sense to create multiple instances of the same template right?

Reply:
@bartfaitamas @richardkapolnai-da note that we only support primitive arguments so far!

Reply:
Yes, what I meant is to support multiple entries in the table, creating multiple contracts from the same template.

Reply:
Done.